### ISSUE TITLE
Implement Follow Path Feature

### ISSUE THREAD

**Issue by R3l3ntl3ss** [fnplus/footsteps-app#15]
Add ability for the user to follow a particular path and the followed paths should be available on their profile.

**Comment by xlogix** [comment]
We could also track the current progress of a learning path and show relevant footsteps in the home page.

**Comment by sanchibansal340** [comment]
Can you please let me know what skillset is required to solve this issue?

**Comment by prajwal714** [comment]
Can you brief the steps required to implement this feature. I would need some help writing the GraphQL queries. 
My Idea is that when the user follows a path, The Path ID will be added to followed_Paths array of the user. Using them we can show the followed paths on their profile.

**Comment by xlogix** [comment]
Yeah, a follow path is basically like the 'watch' feature in Github. We want to show the followed paths on profile and also want to show updates on the path (by the original creator) on the feed page.

**Comment by prajwal714** [comment]
Assign the task to me. @xlogix
